scene/resources/syntax_highlighter.h

```diff
@@ -142,4 +143,36 @@ class CodeHighlighter : public SyntaxHighlighter {
 	Color get_member_variable_color() const;
 };
 
+///////////////////////////////////////////////////////////////////////////////
+
+class GDShaderSyntaxHighlighter : public CodeHighlighter {
+	GDCLASS(GDShaderSyntaxHighlighter, CodeHighlighter)
+
+private:
+	struct BranchRegion {
+		int from_line = -1;
+		int to_line = -1;
+	};
```

Comment:
Can use a `Point2i` rather then adding struct, unless this is for some future changes?